File: Exercicios Diversas Linguagens/C/ex2.c
/*Dado um inteiro positivo n, verificar se n é primo*/

#include <stdio.h>

int main()
{
    int numero, i;
    scanf("%d", &numero);
    for (i = 2; i < numero; i++){
        if (numero%i == 0){
            printf("Nao eh primo\n");
            break;
        }
        if (numero%i != 0 && i+1 == numero){
            printf("Eh primo\n");
        }
    }
    return 0;
}

File: Exercicios Diversas Linguagens/Go/ex2.go
package main
import "fmt"

func main() {
    var n int
    fmt.Println("Digite um numero:")
    fmt.Scanf("%d", &n)
    for i:=2; i<n; i++ {
        if (n%i == 0){
            fmt.Printf("Nao eh primo")
            break
        } else if (n%i != 0) && (i+1 == n){
            fmt.Printf("Eh primo")
        }
    }
}